addons/website_slides/static/tests/tours/slides_course_reviews_reaction_portal.js

```diff
@@ -0,0 +1,33 @@
+/** @odoo-module **/
+
+import { registry } from "@web/core/registry";
+
+/**
+ * This tour test that a portal users can not react to messages
+ */
+registry.category("web_tour.tours").add("course_reviews_reaction_portal", {
+    url: "/slides",
+    steps: () => [
+        {
+            trigger: "a:contains(Basics of Gardening - Test)",
+            run: "click",
+        },
+        {
+            trigger: "a[id=review-tab]",
+            run: "click",
+        },
+        {
+            trigger: "#chatterRoot:shadow .o-mail-Message-textContent:contains(Bad course!)",
+            run: "hover && click",
+        },
+        {
+            trigger: "#chatterRoot:shadow .o-mail-Message .o-mail-Message-actions",
+            run: async () => {
+                const addReactionButton = document.querySelector('#chatterRoot').shadowRoot.querySelector("[title='Add a Reaction']")
+                if (addReactionButton) {
+                    throw new Error("Portal user is able to react");
```

Comment:
I believe the term used here is incorrect, it should be "public user"? As portal user (= logged in) should be able to react.

Reply:
"Public user should not be able to react."